src/utils/DatasetService.js

```diff
@@ -1,57 +0,0 @@
-import {filter, find, flow, getOr, isEmpty, isNil, map} from 'lodash/fp';
-import {DataUseTranslation} from '../libs/dataUseTranslation';
-import {Storage} from '../libs/storage';
-import {USER_ROLES} from '../libs/utils';
-import { DAC } from '../libs/ajax/DAC';
-
-export const DatasetService = {
-
-  /**
-   * Async utility to find all datasets a DAC Chairperson
-   * has access to and populate their data use codes and translations
-   * @returns {Promise<Awaited<dataset>[]>}
-   */
-  populateDacDatasets: async () => {
-    // Find all Chairperson DAC Ids for the user
-    const user = Storage.getCurrentUser();
-    const dacIds = flow(
-      filter(r => r.name === USER_ROLES.chairperson),
-      map('dacId'),
-      filter(id => !isNil(id))
-    )(user.roles);
-    // find all datasets for all DACs the user has access to.
-    const dacDatasets = (await Promise.all(dacIds.map(id => DAC.datasets(id)))).flat();
-    // Enrich datasets with data use translations
-    return await Promise.all(map(async (d) => {
-      const {codeList, translations} = await DatasetService.getDataUseTranslations(d);
-      return Object.assign({}, d, {codeList: codeList, translations: translations});
-    })(dacDatasets));
-  },
-
-  findDatasetPropertyValue: (dataset, propName, defaultVal) => {
-    const defaultValue = isNil(defaultVal) ? '' : defaultVal;
-    return getOr(defaultValue)('propertyValue')(find({propertyName: propName})(dataset.properties));
-  },
-
-  findDatasetPropertyValueList: (dataset, propName, defaultVal) => {
-    const defaultValue = isNil(defaultVal) ? [] : defaultVal;
-    return getOr(defaultValue)('propertyValue')(find({propertyName: propName})(dataset.properties));
-  },
-
-  getDataUseTranslations: async (dataset) => {
-    if (!dataset.dataUse || isEmpty(dataset.dataUse)) {
-      return {
-        codeList: '- -',
-        translations: []
-      };
-    } else {
-      const translations = await DataUseTranslation.translateDataUseRestrictions(dataset.dataUse);
```

Comment:
Reviewers note: this is a particularly expensive operation that we want to try and prune the usages of. The only way around this currently is to use the ES index as the primary source of truth for translated data use objects.